TatianaBelyukina_code/task3/main.py

```diff
@@ -0,0 +1,7 @@
+from split_data import load_data
+from build_models import build_model
+
+x_train, y_train, x_test, y_test = load_data()
+model = build_model()
+
+model.fit(x_train, y_train)
```

Comment:
Consider using `if name == "__main__":` to indicate entry point to program
It also may help you in future if you'll ever try to use `multiprocessing` library